contracts/RelayHub.sol

```diff
@@ -79,8 +82,13 @@ contract RelayHub is IRelayHub {
 
     EIP712Sig private eip712sig;
 
-    constructor () public {
+    constructor (uint256 _gtxdatanonzero) public {
```

Comment:
our first RelayHub constructor param...
- I thought it is "16". you write "68" everywhere
-  what is the source of the value? its left undocumented. 
- should be written in one place, not spread all over the deployment scripts, tests
- you didn't update "singleton/deploy" script